ext/ITensorNetworksEinExprsExt/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,50 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, Algorithm, noncommoninds
```

Comment:
```suggestion
using ITensors: Index, ITensor, Algorithm, @Algorithm_str, noncommoninds
```
should fix the latest CI failures.